doc/user/layouts/shortcodes/create-source/syntax-details.html

```diff
@@ -21,6 +22,26 @@
 
 ## Details
 
+### Materialized source details
+
+By default, sources in Materialize do not retain data from the external
+components to which they connect. Instead, [materialized
+views](../../overview/api-components/#materialized-views) and their attendant
+[indexes](../../overview/api-components/#indexes) retain and manage data. The
+presence of these indexes is necessary to make the views `SELECT`-able; the
```

Comment:
I think this is not strictly speaking correct, though maybe: lemme explain and you decide:

Views are selectable (if this means the same as "QUERYABLE" from `SHOW FULL VIEWS`) if they do not depend on unmaterialized sources. They do not need to have indexes available to be queryable. If they have indexes available they should be very quickly queryable, though!